MobileFilter/sections/specific_web.txt

```diff
@@ -51,6 +51,7 @@ kanasoku.info##.adc_g_ko_sec
 egmnow.com##.pt_mobile_ad_iframe
 egmnow.com##.pt-embed-ad-single-mobile
 m.sondakika.com##.text_reklam
+newtoki.com###hd_pop
```

Comment:
Is not it anti-adblock?

Reply:
no, it's not. anti-adblock doesn't reproducing
![photo_2019-02-05 18 09 26](https://user-images.githubusercontent.com/45171506/52282328-3a706b00-2971-11e9-8715-b77c1aefdb26.jpeg)